From e800d419700a67a344d3b9c347fc6a9e0ede6e3d Mon Sep 17 00:00:00 2001
From: Cheng Lian <lian.cs.zju@gmail.com>
Date: Fri, 1 Aug 2014 16:38:03 -0700
Subject: [PATCH 21/93] PARQUET-13: The `-d` option for `parquet-schema` shouldn't have optional argument

Author: Cheng Lian <lian.cs.zju@gmail.com>

Closes #11 from liancheng/fix-cli-arg and squashes the following commits:

85a5453 [Cheng Lian] Reverted the dummy change
47ce817 [Cheng Lian] Dummy change to trigger Travis
1c0a244 [Cheng Lian] The `-d` option for `parquet-schema` shouldn't have optional argument
---
 .../parquet/tools/command/ShowSchemaCommand.java   |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/parquet-tools/src/main/java/parquet/tools/command/ShowSchemaCommand.java b/parquet-tools/src/main/java/parquet/tools/command/ShowSchemaCommand.java
index 2666a6f..1a8963c 100644
--- a/parquet-tools/src/main/java/parquet/tools/command/ShowSchemaCommand.java
+++ b/parquet-tools/src/main/java/parquet/tools/command/ShowSchemaCommand.java
@@ -44,7 +44,6 @@ public class ShowSchemaCommand extends ArgsOnlyCommand {
     OPTIONS = new Options();
     Option help = OptionBuilder.withLongOpt("detailed")
                                .withDescription("Show detailed information about the schema.")
-                               .hasOptionalArg()
                                .create('d');
     OPTIONS.addOption(help);
   }
-- 
1.7.0.4

